packages/icons/src/docker-grey.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" fill="none"><g clip-path="url(#a)" opacity=".2"><path fill="#B0B1C3" d="M97.936 40.038c-2.252-1.563-7.412-2.148-11.447-1.367-.469-3.906-2.627-7.324-6.38-10.351l-2.158-1.563-1.501 2.246c-1.877 2.93-2.815 7.031-2.534 10.937.094 1.367.563 3.809 1.97 5.957-1.313.781-4.034 1.758-7.599 1.758H.451l-.188.78C-.393 52.343-.393 64.55 7.3 73.924c5.817 7.129 14.45 10.742 25.802 10.742 24.583 0 42.785-11.816 51.323-33.202 3.378.098 10.602 0 14.261-7.324.094-.195.282-.586.939-2.05l.375-.782-2.064-1.27ZM54.682 12.5h-10.32v9.765h10.32V12.5Zm0 11.718h-10.32v9.765h10.32v-9.765Zm-12.197 0h-10.32v9.765h10.32v-9.765Zm-12.197 0H19.967v9.765h10.32v-9.765ZM18.09 35.936H7.77v9.766h10.32v-9.766Zm12.198 0H19.967v9.766h10.32v-9.766Zm12.197 0h-10.32v9.766h10.32v-9.766Zm12.197 0h-10.32v9.766h10.32v-9.766Zm12.198 0H56.559v9.766h10.32v-9.766Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 12.5h100v72.266H0z"/></clipPath></defs></svg>
```

Comment:
Please use view box instead of height and width for sizing of the SVG element

Reply:
yes, updated